R/io-parsers.R

```diff
@@ -615,23 +631,29 @@ parse_mixcr <- function(.filename, .mode) {
   .freq <- "Proportion"
   df$Proportion <- df[[.count]] / sum(df[[.count]], na.rm = TRUE)
 
-  .aa.seq <- IMMCOL$cdr3aa
-  df[[.aa.seq]] <- bunch_translate(df[[.nuc.seq]])
+  .aa.seq.cdr1 <- IMMCOL_EXT$cdr1aa
+  .aa.seq.cdr2 <- IMMCOL_EXT$cdr2aa
+  .aa.seq.cdr3 <- IMMCOL$cdr3aa
+  df[[.aa.seq.cdr1]] <- bunch_translate(df[[.nuc.seq.cdr1]])
+  df[[.aa.seq.cdr2]] <- bunch_translate(df[[.nuc.seq.cdr2]])
```

Comment:
it is not cool to repeat a code several times, better to use some kind of list iterators with purrr::map2();